fabfile.py

```diff
@@ -176,13 +184,52 @@ def deploy(c, target=None, commit=None):
 
 
 @task
-def rollback(c, release=None):
+def rollback(c, target=DEFAULT_TARGET, release=None):
     with Connection(c.deploy.host, user=c.deploy.user, config=c.config) as c:
         setup(c, release=release)
         update(c)
         publish(c)
         finish(c)
 
+# Please add the "target" parameter if you are adding more @task functions
+#  to allow custom targets to be used (regardless if your function itself will use it or not)
+
+
+# Source: https://realpython.com/python-command-line-arguments/#a-few-methods-for-parsing-python-command-line-arguments
+def parseArgs(systemArgs, argStart=1):
```

Comment:
Sir, this is a Wendy's

Can I interest you in [argparse](https://docs.python.org/3/library/argparse.html?highlight=argparse#module-argparse)? ([tutorial](https://docs.python.org/3/library/argparse.html?highlight=argparse#module-argparse)) I find it a lot more readable than custom sys.argv parsing. It's probably more robust to different argument ordering and even provides a help blurb.

Reply:
Thanks
That was was easier than using "sys"

(Can I get fries with that though?)